src/Controls/samples/Controls.Sample/Pages/MainPage.cs

```diff
@@ -341,15 +341,15 @@ IView CreateSampleGrid()
 
 			var bottomLeft = new Label { Text = "Bottom Left", BackgroundColor = Colors.Lavender };
 			layout.Add(bottomLeft);
-			layout.SetRow(bottomLeft, 1);
+			Microsoft.Maui.Controls.Layout2.GridLayout.SetRow(bottomLeft, 1);
```

Comment:
Any particular reason this can't be an instance method too?

Reply:
Might be a conflict of methods due to them both being on the GridLayout type. I think you actually use the static way in all XAML worlds.

But yeah, an instance for all of these will be nice, or maybe an overload for the Add method? `layout.Add(view, row: 1, rowSpan: 3)`

Reply:
`layout.SetRow(bottomRight, 1);` works just fine here. I was just screwing around with the different ways to do it in the sample.